packages/sanity/src/core/components/previews/_common/Media.styled.ts

```diff
@@ -26,17 +25,14 @@ export const MediaWrapper = styled.span<{
     justify-content: center;
 
     & img {
-      position: absolute;
-      left: 0;
-      top: 0;
       width: 100%;
       height: 100%;
       object-fit: contain;
       border-radius: inherit;
     }
 
     & svg {
-      display: block;
+      flex: 1;
```

Comment:
Thank you! @pauloborges

I think changing from `display: block` to `flex: 1` should be enough to fix the issue. Removing `position: relative` and the CSS from `& img` introduces some issues with the absolute positioned `span` that renders the outline around the preview media when it's an `img` (see image):

<img width="346" alt="Screenshot 2023-06-29 at 10 27 43" src="https://github.com/sanity-io/sanity/assets/15094168/f458ef24-320d-4c9c-be9b-f4b08d429955">

Reply:
Hello! Thanks for reviewing it. Do you mind taking this over? You may only need to drop the last commit.